### PR TITLE
For #15561: add comments to issue template

### PR DESCRIPTION
**Pull request by s-ankur**
Documentation change only. The comments are visible only when filing the issue, wont appear on submitted issues.
### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

**Comment by gabrielluong** [comment]
@s-ankur thank you for the contribution! I am handing the review over to @liuche who is the project owner to decide what we would want to do here. I suspect we might want to tweak some of the wording, but I don't have any strong opinions here.

File: .github/ISSUE_TEMPLATE/---bug-report.md [hunk 1]
```diff
@@ -6,8 +6,10 @@ labels: "\U0001F41E bug"
 assignees: ''
 
 ---
-
 ## Steps to reproduce
+[comment]: # (Please do your best to search for duplicate issues before filing a new issue so we can keep our issue board clean)
```
Comment (liuche) [inline .github/ISSUE_TEMPLATE/---bug-report.md:10]: 👍 👍 👍

File: .github/ISSUE_TEMPLATE/---bug-report.md [hunk 2]
```diff
@@ -6,8 +6,10 @@ labels: "\U0001F41E bug"
...
+[comment]: # (Every issue should have exactly bug described in it. Please do not file feedback list tickets as it is difficult to parse them and address their individual points)
```
Comment (liuche) [inline .github/ISSUE_TEMPLATE/---bug-report.md:11]: "have the exact bug and steps to reproduce described in it"

**Review by liuche — APPROVED** [review]
just one nit, could you fix? thank you for this PR @s-ankur!

File: .github/ISSUE_TEMPLATE/---bug-report.md (hunk 2 above)
Comment (s-ankur) [reply .github/ISSUE_TEMPLATE/---bug-report.md:11]: ohh, its supposed to be "exactly one bug". ill correct it

**Pull request merged**